keepalive.go

```diff
@@ -0,0 +1,125 @@
+package ssh
+
+import (
+	"log"
+	"sync"
+	"time"
+)
+
+type SessionKeepAlive struct {
+	clientAliveInterval time.Duration
+	clientAliveCountMax int
+
+	ticker       *time.Ticker
+	tickerCh     <-chan time.Time
```

Comment:
Do we need a separate field for the chan?

Reply:
This is recommendation from https://github.com/coder/ssh/pull/3#discussion_r1221329714